CHANGELOG.md

```diff
@@ -29,6 +29,9 @@ We use the following categories for changes:
 - Log warning when failing to write response to remote read requests [#1180]
 - Fix Promscale running even when some component may fail to start [#1217]
 
+### Removed
+- Remove deprecated flags [#1229]
```

Comment:
Can we refer to some file that contains those flags? Even a `...._test.go` file will also work, since its enough to give users some idea. I recall there was one test for this hence saying.

Reply:
Good point, I'll try to figure something out.